app/templates/index.hbs

```diff
@@ -54,7 +56,8 @@
     </div>
     <div class="layout mt-4">
       <div class="lg:col-4 lg:start-3">
-        <img class="img-fluid" src="https://dummyimage.com/600x400/f47bf6/fff" alt="">
+        <img class="img-fluid" src="/images/home/ember-data.png" alt="" role="presentation" width="636" height="424"
```

Comment:
```suggestion
        <img class="img-fluid" src="/images/home/ember-data.png" alt="" role="presentation" width="636" height="424">
```

Reply:
the 2x png for this graphic is off

Reply:
ah yes I remember you said! I'll fix it now 🎉